dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -259,6 +277,8 @@ dashboard.showProjectHeader = function () {
     });
   });
 
+  $('.project_remix').click(remixProject);
```

Comment:
how does the .project_remix button get added to the dom for the non-minimal project header?